lib/administrate/search.rb

```diff
@@ -24,7 +24,7 @@ def query
         table_name = ActiveRecord::Base.connection.
           quote_table_name(@scoped_resource.table_name)
         attr_name = ActiveRecord::Base.connection.quote_column_name(attr)
-        "lower(#{table_name}.#{attr_name}) LIKE ?"
+        "lower(text(#{table_name}.#{attr_name})) LIKE ?"
```

Comment:
We are writing the SQL keyword `LIKE` in all caps, how would you feel about doing `LOWER` and `TEXT` in all caps as well? I do this in my SQL style so that keywords and column names are clearly separated.

Reply:
Ah yeah, nice. I'll do that, it's much nicer.